[2024-05-10T14:36:55.595+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-10T14:36:55.608+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_seasons.create_db manual__2024-05-10T14:36:51.742460+00:00 [queued]>
[2024-05-10T14:36:55.612+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_seasons.create_db manual__2024-05-10T14:36:51.742460+00:00 [queued]>
[2024-05-10T14:36:55.613+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-10T14:36:55.620+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): create_db> on 2024-05-10 14:36:51.742460+00:00
[2024-05-10T14:36:55.626+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=66) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-10T14:36:55.628+0000] {standard_task_runner.py:63} INFO - Started process 68 to run task
[2024-05-10T14:36:55.628+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'get_seasons', 'create_db', 'manual__2024-05-10T14:36:51.742460+00:00', '--job-id', '163', '--raw', '--subdir', 'DAGS_FOLDER/get_seasons.py', '--cfg-path', '/tmp/tmpxkxx7_w_']
[2024-05-10T14:36:55.629+0000] {standard_task_runner.py:91} INFO - Job 163: Subtask create_db
[2024-05-10T14:36:55.657+0000] {task_command.py:426} INFO - Running <TaskInstance: get_seasons.create_db manual__2024-05-10T14:36:51.742460+00:00 [running]> on host f61d6a8c51ed
[2024-05-10T14:36:55.716+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='get_seasons' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-10T14:36:51.742460+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-10T14:36:51.742460+00:00'
[2024-05-10T14:36:55.717+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-10T14:36:55.717+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-10T14:36:55.717+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/steps/get_id_season.py", line 39, in create_db
    metadata = MetaData()
               ^^^^^^^^
NameError: name 'MetaData' is not defined
[2024-05-10T14:36:55.724+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=get_seasons, task_id=create_db, execution_date=20240510T143651, start_date=20240510T143655, end_date=20240510T143655
[2024-05-10T14:36:55.732+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 163 for task create_db (name 'MetaData' is not defined; 68)
[2024-05-10T14:36:55.761+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-10T14:36:55.775+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-10T14:36:55.777+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
